libs/GraphicECS/SFML/Resources/SoundResource.hpp

```diff
@@ -0,0 +1,47 @@
+/*
+** EPITECH PROJECT, 2022
+** R-Type
+** File description:
+** SoundResources
+*/
+
+#ifndef SOUNDRESOURCES_HPP_
+#define SOUNDRESOURCES_HPP_
+
+#include <SFML/Audio.hpp>
+#include <filesystem>
+#include "Resource/Resource.hpp"
+#include <unordered_map>
+
+namespace ecs
```

Comment:
bad namespace, use graphicECS::SFML::Resources

Reply:
improve in #205 and #206 but yes i change that this afternoon